book/src/spec.md

```diff
@@ -10,33 +10,33 @@ We use Halo2/IPA with [Pasta curves](https://github.com/zcash/pasta) developed b
 Let `C(x; w) ⟶ 0/1` be a circuit with up to `m` gates. The circuit is represented as polynomials over the chosen curve's **scalar field**, following [plonk-ish arithmetization](https://zcash.github.io/halo2/concepts/arithmetization.html). Commitments are generated over the curve's **base field**.
 
 ### 1.2 Elliptic curves
-||Name|Scalar field| Base field|Purpose|Instantiation|
-|-|-|-|-|-|-|
-|$E_I$|Inner curve|$\mathbb{F}_q$|$\mathbb{F}_p$|ECC gadget| [Pallas](https://github.com/zcash/pasta#pallasvesta-supporting-evidence)
-|$E_M$|Main curve|$\mathbb{F}_p$|$\mathbb{F}_q$|Action and VP circuits| Vesta|
-|$E_O$|Outer curve|$\mathbb{F}_q$|$\mathbb{F}_p$|Accumulation circuit| Pallas|
+|Name|Base field| Scalar field|Purpose|Instantiation|
+|-|-|-|-|-|
+|$E_p$|$\mathbb{F}_p$|$\mathbb{F}_q$|ECC gadget, Accumulation circuit| [Pallas](https://github.com/zcash/pasta#pallasvesta-supporting-evidence)
```

Comment:
I'm not sure if this unification is helpful. Accumulation circuit and ECC gadget are defined over the same circuit only because Halo2 happened to have curve cycle, but they are not the same conceptually. I think having inner curve and outer curve is helpful.

Reply:
Most of the specs are conditioned by us using Halo2, this is not an exception. We have no reference to outer or inner curves specifically in any of our documentation

Reply:
I still find it useful to keep inner and outer curves separately, even if in Halo2 it is sort of the same. Maybe ask @XuyangSong for the third opinion